governance/tests/test-sli-boundary.sh

```
@@ -0,0 +1,66 @@
+  && ok "sli-report 周报模板含 env_face（数据源=env-ledger）" || bad "周报未接环境面"
+
+# ---- AC-8d：责任边界文件四节齐备 ----
+DOC="$DIR/docs/slo-boundary.md"
+[[ -f "$DOC" ]] && ok "docs/slo-boundary.md 落盘" || bad "责任边界文件缺失"
+for SEC in "SLO 定义" "值班范围" "破线升级路径" "break-glass"; do
+  grep -q "## .*$SEC" "$DOC" && ok "四节齐备：$SEC" || bad "缺节：$SEC"
+done
+
+# SLO 定义含骨架期指标（SLO-1 收敛/SLO-2 新鲜度——与 env_face 指标对应）
+grep -q "SLO-1" "$DOC" && grep -q "SLO-2" "$DOC" && grep -q "convergence" "$DOC" \
+  && ok "SLO-1/2 与 env_face convergence 对应（定义↔数据源）" || bad "SLO 定义与指标脱节"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)